src/ValueObjects/TransformationV2.php

```diff
@@ -94,11 +94,16 @@ public function getBackend(): string
         return $this->backend;
     }
 
-    private function renameInputMappingKeys(array $inputMappingTable): array
+    private function renameInputMappingKeys(array $inputMappingTable, ?string $keyPrefix = null): array
```

Comment:
ten `$keyPrefix` sa niekde pouziva, kde?

Reply:
jj máš pravdu už jsme to dělali, ale nebyl na to task v Jire, tak jsem ho vytvořil

Reply:
tady v tom PR jsem doplnil nahrazení `datatypes`

![image](https://user-images.githubusercontent.com/12143866/113986068-1eb4c800-984d-11eb-9a34-ff77e98b9244.png)

a childů `column` a `convertEmpty...`

Reply:
super. uz je to jasne.